com.github.kagami.boram.yml

```diff
@@ -0,0 +1,241 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
are you sure it supports wayland?

Reply:
I've tested it on Ubuntu 22.04 with Wayland and it worked properly.

Reply:
but it uses gtk2.

Reply:
You're absolutely correct. I built it again from scratch and not having the `--socket=x11` broke it, so I've removed `--socked-wayland` and `--socket=fallback-x11`.